src/nxt_sprintf.c

```diff
@@ -115,6 +115,7 @@ nxt_vsprintf(u_char *buf, u_char *end, const char *fmt, va_list args)
     static const u_char  hexadecimal[16] = "0123456789abcdef";
     static const u_char  HEXADECIMAL[16] = "0123456789ABCDEF";
     static const u_char  nan[] = "[nan]";
```

Comment:
BTW, this is not part of this patch, but I noticed some undefined behavior here.  We should be using the `nxt_` prefix.  `nan` is a standard libm function, and we're including `<math.h>`, so that is another Undefined Behavior.  See <https://man7.org/linux/man-pages/man3/nan.3p.html>.

Reply:
See POSIX <https://pubs.opengroup.org/onlinepubs/9699919799/functions/V2_chap02.html>:


> The following identifiers are reserved regardless of the inclusion of headers:
> 
> ...
> 3.
>     All identifiers in the table below are reserved for use as identifiers with external linkage. Some of these identifiers do not appear in this volume of POSIX.1-2017, but are reserved for future use by the ISO C standard.
> ...
> 
> No other identifiers are reserved.
> 
> 
> ...
> nan
> ...